src/config.js

```diff
@@ -112,7 +112,7 @@ global.config = {
     maxRooms: 8,
     cpuPerRoom: 13, // Necessary CPU per room, prevent claiming new rooms
     revive: true,
-    maxDistance: 17,
+    maxDistance: 10,
```

Comment:
Why?

Reply:
my creepy died before they reach 17, so why seach for far away places, that aren't rechable

Reply:
Mind to write this as a comment to the line, so that we know in the future why we went in this direction?